CMakeLists.txt

```diff
@@ -143,6 +143,9 @@ list(APPEND CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_PREFIX}/ll
 option(ENABLE_HIP_WORKAROUNDS Off)
 set(MIOPEN_INSTALL_CXX_HEADERS Off CACHE BOOL "Install MIOpen's C++ header interface")
 
+set_var_to_condition(MIOPEN_OFFLINE_COMPILER_PATHS_V2_DEFAULT FALSE)
+option(MIOPEN_OFFLINE_COMPILER_PATHS_V2 "Use rocm-core to find offline GPU compiler" ${MIOPEN_OFFLINE_COMPILER_PATHS_V2_DEFAULT})
```

Comment:
It is much better to use `cmake_dependent_option` here and make it `OFF` by default if compiling on Windows. Additionally, the command will "hide" the option from the available options list, making it impossible to use on Windows.

Reply:
@apwojcik See my previous comment, https://github.com/ROCm/MIOpen/pull/2694#discussion_r1488728293